proposals/lip-0061.md

```diff
@@ -140,7 +156,9 @@ certificateSchema = {
         "height",
         "timestamp",
         "stateRoot",
-        "validatorsHash"
+        "validatorsHash",
```

Comment:
So if I understant correctly, rather than having a new type 'unsigned certificate', you set `aggregationBits` and `signature` to empty bytes. I guess this is fine in this case, since there is no concept of ID and the signature should still be fine, but one drawback is that we can not impose the length of the signature in the schema (which we do in other places) and that we are changing the protocol.

Reply:
As discussed, I now introduce an unsigned certificate schema and type.